backend/internal/adapter/persistence/sqlc_convert.go

```
@@ -0,0 +1,57 @@
+
+import (
+	"time"
+
+	"github.com/jackc/pgx/v5/pgtype"
+)
+
+// GORM時代のmodelは非ポインタのstring/int/boolフィールドにNULL許容カラムを載せていたため、
+// sqlc(pgx)側のポインタ/pgtype表現との間で変換が繰り返し必要になる。ここに集約する。
+
+// fromStringPtr はNULL許容カラムの *string を model の string へ変換する（NULLは空文字）。
+func fromStringPtr(p *string) string {
```

golangci-lint (2.13.2)

[error] func fromStringPtr is unused

(unused)